File: TempConverter/C#/TempConverter/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace TempConverter
{
    public partial class Form1 : Form
    {
        double VarTmpC; // Creating variables
        double VarTmpF;
        double VarTmpK;

        public Form1()
        {
            InitializeComponent();

        }
        private void TempC_TextChanged(object sender, EventArgs e)
        {
            String input = TempC.Text;
            if (double.TryParse(input, out VarTmpC)) // Checks if the input is a number
            {
                VarTmpF = (9 * VarTmpC / 5) + 32; // Formula for ºC -> ºF
                VarTmpK = VarTmpC + 273.15; // Formula for ºC -> K
                TempK.Text = VarTmpK + ""; // Converts double variable to a textbox
                TempF.Text = VarTmpF + ""; // There is a nicer way to do this, but this was easier
            }
            else // Triggers an error when the user enters a non-number value
            {
                TempK.Text = "ERROR";
                TempF.Text = TempK.Text;
            }
        }
    }
}

File: TempConverter/VisualBasic/TempConverter/TempConverter/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) 

    End Sub

    Private Sub TempC_TextChanged(sender As Object, e As EventArgs) Handles TempC.TextChanged
        If IsNumeric(TempC.Text) = True Then
            TempK.Text = TempC.Text + 237.15
            TempF.Text = (TempC.Text * 9 / 5) + 32
        Else
            TempK.Text = "ERROR"
            TempF.Text = TempK.Text
        End If
    End Sub
End Class
